frontend/src/app/features/work-packages/components/wp-table/context-menu-helper/wp-context-menu-helper.service.ts

```diff
@@ -103,13 +103,16 @@ export class WorkPackageContextMenuHelperService {
 
   public getIntersectOfPermittedActions(workPackages:any) {
     const bulkPermittedActions:any = [];
+
     const possibleActions = this.BULK_ACTIONS.concat(this.HookService.call('workPackageBulkContextMenu'));
-    const permittedActions = _.filter(possibleActions, (action:any) => _.every(workPackages, (workPackage:WorkPackageResource) => this.getAllowedActions(workPackage, [action]).length >= 1));
+    const permittedActions = _.filter(possibleActions, (action:any) => _.every(workPackages, (workPackage:WorkPackageResource) => this.isActionAllowed(workPackage, action)));
```

Comment:
wait, shouldn't this be part of the previous pr/commit/branch?

Reply:
ah wait, this is the PR about fixing the permitted actions? Don't we have that already sitting as a PR on opf/openproject?

Reply:
Oh I combined both of them, but maybe it was not a good idea.
I will change the title of this Pr, revert it to only take care of the allowed actions check, and maybe add the further `icon` changes to your branch, if that's ok?

Reply:
At the moment this PR is pointing to the "Allow plugins to link into the WP bulk context menu".
But if we get the one on the Open Project repo merged, we won't need this one.